src/WhiteLibrary/keywords/application.py

```diff
@@ -26,6 +27,31 @@ def launch_application(self, sut_path, args=None):
         else:
             self.state.app = Application.Launch(sut_path)
 
+    def _attach_application(self, sut_identifier):
```

Comment:
This could take a timeout argument and if the value is None, use the attach_application_timeout?